eclass/kernel-build.eclass

```diff
@@ -20,6 +20,17 @@
 # the kernel and installing it along with its modules and subset
 # of sources needed to build external modules.
 
+# @ECLASS_VARIABLE: KV_FULL
+# @USER_VARIABLE
+# @DESCRIPTION
+# A string containing the full kernel release version,
+# e.g. '6.9.6-gentoo-dist'. This is used to ensure
+# consistency between the kernel's release version and
+# Gentoo's tooling. Must be set by the inheritor before
+# calling kernel-build_src_compile(). If not set,
+# kernel-build_src_compile() will use the built kernel's
+# release file.
```

Comment:
Ok, this is confusing and I think it is wrong too.

1. `@USER_VARIABLE` is something you set in `make.conf`.
2. Is "inheritor" the ebuild or what?
3. Either it "must" be set, or it is set automatically. You can't say it must be set, then say it's fine when it's not set.

Reply:
Ah sorry, the documentation around `@USER_VARIABLE` was a little confusing to me. Wording also changed to clarify how to use the variable.